hummingbot/connector/exchange/probit/probit_user_stream_tracker.py

```diff
@@ -38,14 +33,16 @@ def __init__(self,
                  trading_pairs: Optional[List[str]] = None,
                  domain: str = "com",
                  shared_client: Optional[aiohttp.ClientSession] = None):
-        super().__init__()
         self._shared_client = shared_client
         self._domain: str = domain
         self._probit_auth: ProbitAuth = probit_auth
         self._trading_pairs: List[str] = trading_pairs or []
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
+        super().__init__(data_source=ProbitAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by super().__init__. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.